Gems/GradientSignal/Code/Include/GradientSignal/Components/PerlinGradientComponent.h

```diff
@@ -61,7 +61,7 @@ namespace GradientSignal
 
         PerlinGradientComponent(const PerlinGradientConfig& configuration);
         PerlinGradientComponent() = default;
-        ~PerlinGradientComponent() = default;
+        virtual ~PerlinGradientComponent() = default;
```

Comment:
The base AZ::Component destructor, so this destructor is also virtual, therefore the `virtual` keyword isn't needed.